src/MagnumPlugins/WavAudioImporter/WavHeader.h

```diff
@@ -31,30 +31,59 @@
 
 #include "Magnum/Types.h"
 
+#define WAVE_FORMAT_PCM 0x0001
```

Comment:
I'd also prefer a (documented) enum to avoid polluting the global namespace. No need to create the Debug operator for it, since this is used only internally (unless you find a good reason to have it, e.g. usage in error messages). Also, since the `WavHeader.h` is internal header, not exposed to plugin users, there's no need to have them internal to the source file.

I'll check with `littleEndianInPlace()`, it should work on enums. If not, I'll make the required changes, because this is a valid use case.